InventoryAdminUi/Test/Mftf/Test/StorefrontOrderCreatedWithConfigurableProductWithSwatchAttributeTest.xml

```diff
@@ -118,10 +118,10 @@
             <argument name="product" value="ConfigurableMsiProduct"/>
         </actionGroup>
 
-        <seeElement selector="//div[contains(@id, 'option-label-visual_swatch_')][@option-label='{{visualSwatchColor1.name}}']" stepKey="seeRedOption1"/>
-        <seeElement selector="//div[contains(@id, 'option-label-visual_swatch_')][@option-label='{{visualSwatchColor2.name}}']" stepKey="seeBlueOption1"/>
+        <seeElement selector="//div[contains(@id, 'option-label-visual_swatch_')][@data-option-label='{{visualSwatchColor1.name}}']" stepKey="seeRedOption1"/>
+        <seeElement selector="//div[contains(@id, 'option-label-visual_swatch_')][@data-option-label='{{visualSwatchColor2.name}}']" stepKey="seeBlueOption1"/>
 
-        <click selector="div[id*='option-label-visual_swatch_'][option-label='{{visualSwatchColor1.name}}']" stepKey="clickOnRedOption1"/>
+        <click selector="div[id*='option-label-visual_swatch_'][data-option-label='{{visualSwatchColor1.name}}']" stepKey="clickOnRedOption1"/>
```

Comment:
```suggestion
        <click selector="div[id*='option-label-visual_swatch_'][@data-option-label='{{visualSwatchColor1.name}}']" stepKey="clickOnRedOption1"/>
```

Reply:
No. This will cause the test to fail

Reply:
@engcom-Echo That is OK. I haven't noticed that this one is not Xpath, just CSS selector! I'm sorry.

Reply:
This is css selector, it shouldn't contains `@`